sys-power/thermald/thermald-2.4.5-r1.ebuild

```diff
@@ -22,16 +22,16 @@ RDEPEND="
 	sys-power/upower
 	sys-apps/dbus:="
 DEPEND="${RDEPEND}
-	dev-util/gtk-doc
+	doc? ( dev-util/gtk-doc )
 	dev-util/glib-utils"
 
 S=${WORKDIR}/thermal_daemon-${PV}
 DOCS=( thermal_daemon_usage.txt README.txt )
-
 src_prepare() {
 	sed -i -e "/group=/s/power/wheel/g" \
 		data/org.freedesktop.thermald.conf || die
 
+	use doc || PATCHES+=("${FILESDIR}/${PN}-no-docs.patch")
```

Comment:
sorry, I didn't review the patch carefully..

you should introduce a "--enable-doc" option for upstream, and also avoid the conditional patch here

Reply:
> sorry, I didn't review the patch carefully..
> 
> you should introduce a "--enable-doc" option for upstream, and also avoid the conditional patch here

Thank you for your review and suggestion. After some more consideration I'm afraid I'll have to NACK that.

I actually don't know autotools and friends to a noteworthy degree, I don't think upstream will be interested, and my goal was mainly to get rid of that dependency on my Gentoo installations (it seemed most if not all other users of gtk-doc are conditional).

So, feel free to close/reject this PR, I'm happy to maintain it in my local overlays as-is.

Reply:
> > sorry, I didn't review the patch carefully..
> > you should introduce a "--enable-doc" option for upstream, and also avoid the conditional patch here
> 
> Thank you for your review and suggestion. After some more consideration I'm afraid I'll have to NACK that.
> 
> I actually don't know autotools and friends to a noteworthy degree, I don't think upstream will be interested, and my goal was mainly to get rid of that dependency on my Gentoo installations (it seemed most if not all other users of gtk-doc are conditional).
> 
I'll ping upstream first about the idea of adding optional 'doc' configuration

> So, feel free to close/reject this PR, I'm happy to maintain it in my local overlays as-is.
generally, trying to maintain patches downstream is kind of painful, it will easily become broken..
for now, please feel free to maintain it as in your local overlay, thanks